src/redux/userManager/reducer.js

```diff
@@ -15,6 +15,7 @@ const getDefaultState = () => ({
     lastName: null,
     isLoading: true,
   },
+  everShowDataDisclaimer: true,
```

Comment:
Do we need this property in the reducer?

Reply:
Good catch